tests/unit/packet/PublishTest.php

```diff
@@ -225,65 +244,67 @@ public function testParseWithQos($qos, $bit)
     {
         $input =
             chr($bit) .
-            chr(4) .
+//            chr(4) .
```

Comment:
OK, I'm starting to understand what all these numbers mean. I'll get back to you...